recipe-generator/src/main/resources/templates/js/ingredient.liquid

```diff
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
     {%- assign superclass = 'Ingredient' -%}
 {%- endif -%}
 {%- assign ingredientName = ingredient.name | append:options.ingredientPostfix -%}
-const {{superclass}} = require("recipe-ts-runtime").{{superclass}};
+const {{superclass}} = require("{{options.runtimeLibrary}}").{{superclass}};
```

Comment:
Is this needed? Why not just make the ts-runtime be the js-runtime? TS should really just be the JS + .d.ts for compile-time.

Note that I really have no idea what's going on so I might be completely wrong...

Reply:
We need this because let's say someone wants to generate ingredients in JavaScript and write recipe tests in JavaScript. They're going to only add `recipe-js-runtime` to their dependencies. However, this template is reused when we're generating typescript (generated js plus ts types), so in that case the user will have had a dependency on `recipe-ts-runtime` instead. The generated ingredients need to include the runtime your project is actually using.

Reply:
But typescript doesn't exist at "runtime", so wouldn't the js+ts libraries be the exact same?